### PR TITLE
RS-119: Optimal Image Storage Solutions for ROS-Based Computer Vision

### PR DESCRIPTION
**Pull request by AnthonyCvn**
New blog post about integrating ReductStore with ROS for robotic applications.

### CODE REVIEW

**Review by atimin — CHANGES_REQUESTED** [review]
The article has a good and clear structure and it was interesting to read.  However, I suggest the following changes:
1. In reduct-py 1.9 we improved user experience with timestamps. Could you use the new API in the example so that we show the current approach? 
2. Could we add something about how to read or export the data? Maybe we can show how to store a raw image in JPEG format and then export the data with `reduct-clI`?

**Comment by AnthonyCvn** [comment]
@atimin I made quite some changes to the ROS demo repo and blog for your review.

File: blog/2024-03-15-ROS-integration/index.md [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+
+## Best practices
+
+There are several best practices to consider when integrating ReductStore. Here are a few to keep in mind:
+
+- Use non-blocking operations to avoid blocking the main thread of the ROS 2 node. This ensures that the node can continue processing other tasks while waiting for data to be stored.
+- Serialize data before storing it in ReductStore in a cross-platform binary format to ensure compatibility with different systems and programming languages.
+- Create a ReductStore bucket with a [**FIFO**](<https://www.reduct.store/docs/how-does-it-work#bucket>) quota to prevent disk overwriting in the future.
+- Use token authentication to protect your data. You can generate an access token using either the [**Web Console**](<https://github.com/reductstore/web-console>) or the [**CLI client**](<https://cli.reduct.store/>).
+- Use the [**Web Console**](<https://github.com/reductstore/web-console>) or [**ReductCLI**](<http://cli.reduct.store/>) for data replication or backup purposes.
```
Comment (atimin) [inline blog/2024-03-15-ROS-integration/index.md:271]: use the web console or reductcli to set up data replication between two instances
use reductcli for manual replication or backup

**Review by atimin — APPROVED** [review]
One remark in the Best Practices section and we can publish

**Pull request merged**